### PR TITLE
Add optional uri

### PR DESCRIPTION
**Pull request by nschairer**
Adding optional URI param so you can use HMR for development (ex: vue dev server)

### CODE REVIEW

File: include/vstwebview/webview_pluginview.h [hunk 1]
```diff
@@ -39,7 +39,9 @@ class WebviewPluginView : public Steinberg::Vst::EditorView {
   WebviewPluginView(Steinberg::Vst::EditController *controller,
                     const std::string &title,
                     const std::vector<vstwebview::Bindings *> &bindings,
-                    Steinberg::ViewRect *size = nullptr);
+                    Steinberg::ViewRect *size = nullptr,
+                    const std::string uri = nullptr
+                    );
```
Comment (rdaum) [inline include/vstwebview/webview_pluginview.h]: nit: ); should probably be after nullptr. Try running through clang-format with --style Google

File: include/vstwebview/webview_pluginview.h [hunk 2]
```diff
@@ -39,7 +39,9 @@ class WebviewPluginView : public Steinberg::Vst::EditorView {
   WebviewPluginView(Steinberg::Vst::EditController *controller,
                     const std::string &title,
                     const std::vector<vstwebview::Bindings *> &bindings,
-                    Steinberg::ViewRect *size = nullptr);
+                    Steinberg::ViewRect *size = nullptr,
+                    const std::string uri = nullptr
```
Comment (rdaum) [inline include/vstwebview/webview_pluginview.h]: Might want to consider std::optional<std::string> instead.

File: include/vstwebview/webview_pluginview.h [hunk 3]
```diff
@@ -58,6 +60,7 @@ class WebviewPluginView : public Steinberg::Vst::EditorView {
   const std::string &title_;
   std::unique_ptr<vstwebview::Webview> webview_handle_;
   std::vector<vstwebview::Bindings *> bindings_;
+  std::string uri;
```
Comment (rdaum) [inline include/vstwebview/webview_pluginview.h]: uri_

File: include/vstwebview/webview_pluginview.h (hunk 2 above)
Comment (nschairer) [reply include/vstwebview/webview_pluginview.h]: I switched it to default to an empty string ( uri = {} ), more or less the same right?

**Pull request merged**